content/messaging/index.mdx

```diff
@@ -88,6 +88,30 @@ To learn about differences between these two API types, see the comparison below
 
 If you're not sure, which implementation to choose, we suggest reading about particular APIs. You'll find the **When to use RTM/Web API** section in each document - refer to the table above for links.
 
+## API Versioning
+
+LiveChat Messaging API doesn't follow <a href="hhttps://semver.org/" target="_blank">semantic versioning</a>. We suggest you learn our system and terminology:
+
+- **stable**
+  
+  The latest stable version we encourage developers to use. We don't introduce any new functionalities or breaking changes - only bug fixes are allowed.
+
+- **developer preview (dev preview)**
```

Comment:
perhaps just one name is enough, wdyt?

Reply:
I'd like to have this abbreviation explain somewhere in the docs. It's quite obvious that developer preview = dev preview, but since we used the two options interchangeably, let's keep it here for the sake of clarity.

Reply:
ok